libs/surfaces/console1/console1.cc

```diff
@@ -219,159 +258,179 @@ Console1::connect_internal_signals ()
 	  console1_connections, MISSING_INVALIDATOR, [] () { DEBUG_TRACE (DEBUG::Console1, "VerticalZoomOut\n"); }, this);
 }
 
+void
+Console1::notify_session_loaded ()
+{
+	DEBUG_TRACE (DEBUG::Console1, "************** Session Loaded() ********************\n");
+	create_strip_inventory ();
+	connect_internal_signals ();
+	/*if (session) {
```

Comment:
I tried a lot to find out if there is already a valid session - but nothings seems to wor so far. Do you have any idea where we have a reliable session?

Reply:
Control protocols are instantiated for a session, they do not exist across sessions. So, if you control protocol has been instantiated, there is a valid session (it was passed into the constructor).

Reply:
Thanks Paul, but why does first_selected_stripable() not return a stripable even if one is already selected here? That's what I wanted to achieve here....

Reply:
selection probably hasn't happened yet.

Reply:
;-) Ok, but is there any way to get the initial selction? stripable_selection_changed() does not seem to be called initially.
If I manually select another stripable after loading, the method is called.